components/layout/Footer.jsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import Anchor from "../ui/Anchor";
+import anchors from "../../data/footerAnchor";
+
+export default function Footer() {
+  return (
+      <footer className="flex mx-10 text-white flex-col md:flex-row  md:justify-between  mb-12 mt-17.25  md:mt-31.75 xl:mt-33.75">
```

Comment:
This mx-10 only applies on desktop. Add xl to mitigate it from affecting mobile as it is making the content fall too inside that it should normally be.

![image](https://user-images.githubusercontent.com/90575886/211771501-8c0b5d5c-4458-445e-925c-67e27d475fd1.png)

Footer content should be on the same like as the word "cooperate".